app/src/main/res/layout/view_page_schedule.xml

```diff
@@ -39,4 +39,12 @@
     android:layout_height="wrap_content"
     android:layout_gravity="center" />
 
+  <TextView
+    style="@style/Schedule.EmptyView"
```

Comment:
cmd-alt-l please, id goes first, then style (if you're using the right settings it'll fix it for you)

Reply:
671196f